platform-server-kernel/src/main/java/uno/cod/platform/server/core/service/OrganizationService.java

```diff
@@ -52,21 +54,19 @@ public void createFromDto(OrganizationCreateDto dto, String owner) {
         organizationMembershipRepository.save(organizationMembership);
     }
 
-    public OrganizationShowDto findById(UUID id){
+    public OrganizationShowDto findById(UUID id) {
         return OrganizationMapper.map(organizationRepository.findOne(id));
     }
 
-    public OrganizationShowDto findUserAdminOrganization(String username){
+    public List<OrganizationMembershipShowDto> findUserAdminOrganizations(String username) {
         User user = userRepository.findByUsernameOrEmail(username, username);
-        for(OrganizationMembership member: user.getOrganizationMemberships()){
-            if(member.isAdmin()){
-                return OrganizationMapper.map(member.getKey().getOrganization());
-            }
+        if (user.getOrganizationMemberships() == null || user.getOrganizationMemberships().size() == 0) {
+            return null;
```

Comment:
Also, `user.getOrganizationmemberships().size() == 0` does not have to be handled explicitly. The stream below will yield an empty list in this case.